src/styles/blocks/container.scss

```diff
@@ -0,0 +1,47 @@
+.cotainer {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__block {
+    height: $blockHeight;
+    min-width: $blockHeight;
+    text-align: center;
+    line-height: $blockHeight;
+
+    &--1 {
+      background-color: rgb(255, 0, 0);
+    }
+
+    &--2 {
+      background-color: rgb(204, 0, 0);
+    }
+
```

Comment:
If we need 4/5 of red and 1/5 of black according to the task, I think `rgb(80%, 0, 0)` solution is more understandable where we got this number.